src/test/java/seedu/address/logic/commands/ViewCommandTest.java

```diff
@@ -0,0 +1,91 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.commons.core.Messages.MESSAGE_NAME_NOT_FOUND;
+import static seedu.address.commons.core.Messages.MESSAGE_PERSON_FULL_VIEW;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandFailure;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.testutil.TypicalPersons.getTypicalAddressBook;
+
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.person.NameIsKeywordsPredicate;
+
+public class ViewCommandTest {
+    private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+    private Model expectedModel = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+
+    @Test
+    public void equals() {
+        NameIsKeywordsPredicate firstPredicate =
+                new NameIsKeywordsPredicate(Collections.singletonList("alice pauline"));
+        NameIsKeywordsPredicate secondPredicate =
+                new NameIsKeywordsPredicate(Collections.singletonList("benson meier"));
```

Comment:
Same comment as the other PR, you should try to declare them as static final Strings. Usually they are stored in CommandTestUtil, but you can declare them here if they are only applicable to ViewCommand